bin/arcgis/SMODERP2D.pyt

```diff
@@ -8,7 +8,6 @@ py3 = sys.version_info[0] == 3
 if py3:
     from importlib import reload
 
-sys.path.insert(0, os.path.join(os.path.dirname(__file__), "..", ".."))

```

Comment:
I am not aware of any easy way to install smoderp2d Python package in ArcGIS Pro.  This change will complicate SMODERP2D usage on this platform. Must be discussed.

Reply:
Solved by [564629d](https://github.com/storm-fsv-cvut/smoderp2d/pull/325/commits/564629d9a0c2948401e3258cc137c1337ef067d6)